### ISSUE TITLE
Docking window leaves a small gap

### ISSUE THREAD

**Issue by v71** [ocornut/imgui#6805]
When docking a window repeatedly it happens that a small black gap is left unfilled between windows, as you can see in the screen shot.
What I am doing wrong ?

![Screenshot (331)](https://github.com/ocornut/imgui/assets/3400295/8eca8342-ce25-4290-8883-e26f5533f47b)

**Comment by ocornut** [comment]
> What I am doing wrong ?

We can’t know, since you aren’t posting code nor much detaiils. Please read Issues/Contributing Guidelines and amend your post with more details. Thank you !

**Comment by v71** [comment]
Hello I am just using the code from the example docking branch, I have noticed that this happens when I try to resize a window up to the point it reaches the title bar of the 'parent' window.

**Comment by ocornut** [comment]
Obviously the issue doesn't happen in example docking branch so you'll need to narrow down your issue if you want help.
I don't understand your explanation. See [Glossary](https://github.com/ocornut/imgui/wiki/Glossary) and [Contributing Guidelines](https://github.com/ocornut/imgui/blob/master/docs/CONTRIBUTING.md).

**Comment by v71** [comment]
The 'issue' does appear in the docking demo, I noticed that probably this is not a bug but a 'feature'
![Screenshot (332)](https://github.com/ocornut/imgui/assets/3400295/62c9de98-3700-42a6-bf5b-fc19d44801aa)

**Comment by ocornut** [comment]
I don't quite understand what I am seeing, and I cannot recreate in the demo if I run one of our example.

Again, you are making this more difficult and requiring more back & forth, by not providing a minimum verifiable repro and not providing information requested by the issue template and issue guidelines, so I can only make guesses. I would appreciate that you respect my time a little more there.

**Comment by v71** [comment]
I should put all the docking demo source code then , if you compare the two images , its evident what I am referring , anyways, I don't want to waste your precious time here.

**Comment by ocornut** [comment]
OK so I understand you are talking about the central node that you didn't dock anything into?
The grey area here?

![image](https://github.com/ocornut/imgui/assets/8225057/7dfea5c2-0d5d-443c-a130-8bb4140b7fa1)

It's curious that you made this harder to understand by making it as small as possible in your screenshot, when by default it is so big.

If you intently always dock on the side of space and never into it, it's going to be left empty? 
I'm not sure what you are asking? If you don't want this space dock into it instead of splitting it in half?

**Comment by ocornut** [comment]
If you always dock on the side you are splitting the space in two and leaving an empty space intentionally.
Dock in the center to fill this space:
![image](https://github.com/ocornut/imgui/assets/8225057/da70b4be-f16a-43d6-8e6a-aac98b66b523)

**Comment by v71** [comment]
In my example there are 3 windows, a central node where the rendering happens, and two other windows , and yes, you are probably right, when i dock using the center tab the issue goes away, is this intended to be so ?

**Comment by ocornut** [comment]
You have a bunch of code using `imgui_internal.h` that you omitted to share, and simultaneously said _"The 'issue' does appear in the docking demo"_  ???
Look I'm sorry but you are behaving in a stupid manner, never provided information required by template and guidelines.

Try to use Docking manually with the mouse and you'll understand how this maps to your function calls. If you ask to leave the center node empty, the center node will be left empty. If you fill the center node, it won't be empty.
